etc/output_files.yaml

```diff
@@ -230,4 +230,154 @@ project_process:
 
     relatedness_output_gel:
         location: ['']
-        basename: '{project_id}.relatedness_output.gel'
\ No newline at end of file
+        basename: '{project_id}.relatedness_output.gel'
+
+gatk4_human_var_calling:
+    gvcf:
+        location: ['gatk4']
+        basename: '{user_sample_id}.g.vcf.gz'
+
+    gvcf_tbi:
+        location: ['gatk4']
+        basename: '{user_sample_id}.g.vcf.gz.tbi'
+
+    vcf:
+        location: ['gatk4']
+        basename: '{user_sample_id}_vqsr.vcf.gz'
+        new_name: '{user_sample_id}.vcf.gz'
```

Comment:
This would make VQSR filtering implicit when delivering to the user. Are we sure we want this?

Reply:
I think there is a strong argument for having a single `vcf` being output by each pipeline. Less so to have the name of the file reflect the filtering process.
I the file name are not consistent then it will make the data delivery more complicated.

Reply:
As discussed, the renaming can stay as it is - there's enough information already in the pipeline name and file contents, and it's simpler to say there's only one deliverable VCF